modules/appeals_api/spec/fixtures/valid_200996_v2_extra.json

```diff
@@ -1,154 +1,153 @@
 {
```

Comment:
Sorry for the long commit change. This file's indentation appears to have changed. The only actual content change is at line#31 where I've inserted the key `name` to correctly form the representative's name object.